packages/twenty-front/src/modules/advanced-text-editor/components/__stories__/AdvancedTextEditor.stories.tsx

```diff
@@ -438,3 +441,16 @@ export const WithLists: Story = {
     });
   },
 };
+export const WithSlashCommand: Story = {
```

Comment:
Probably no need for two additional stories but 1 is enough and enableSlashCommand: true, can be added to the main one